src/main/java/com/google/codeu/data/TeamDatastore.java

```diff
@@ -0,0 +1,61 @@
+package com.google.codeu.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import java.util.UUID;
+import com.google.codeu.data.Team;
+
+public class TeamDatastore {
+
+    private DatastoreService teamDatastore;
+
+    // Create runs as though all the team information has been set already.
+    public Team create(Team team) {
+        Entity teamEntity = new Entity("Team");
```

Comment:
I think the Entity constructor takes in a second value for the key(take a look at the storeMessage method in Datastore.java).
 ```suggestion
        Entity teamEntity = new Entity("Team", team.getTeamId().toString());
```

Then you won't have to do:
```suggestion
        teamEntity.setProperty("teamID", team.getTeamID());
```